makefiles/arpoison.mk

```diff
@@ -0,0 +1,57 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += arpoison
+ARPOISON_VERSION := 0.7
+DEB_ARPOISON_V   ?= $(ARPOISON_VERSION)
+
+arpoison-setup: setup libnet
+	wget -q -nc -P$(BUILD_SOURCE) http://www.arpoison.net/arpoison-$(ARPOISON_VERSION).tar.gz
+	$(call EXTRACT_TAR,arpoison-$(ARPOISON_VERSION).tar.gz,arpoison-$(ARPOISON_VERSION),arpoison)
+	mkdir -p $(BUILD_STAGE)/arpoison/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/usr/local/bin/
+	mkdir -p $(BUILD_WORK)/linking/
+	ln -sf $(BUILD_STAGE)/libnet/usr/lib/libnet.a $(BUILD_WORK)/linking/libnet.a
+
+ifneq ($(wildcard $(BUILD_WORK)/arpoison/.build_complete),)
+arpoison:
+	@echo "Using previously built arpoison."
+else
+arpoison: arpoison-setup
+	# arpoison.o
+	$(CC) $(CFLAGS) \
+		-c -o $(BUILD_WORK)/arpoison/arpoison.o -x c \
+		$(BUILD_WORK)/arpoison/arpoison.c\
+	
+	# arpoison
+	$(CC) $(CFLAGS) \
+		-o $(BUILD_WORK)/arpoison/arpoison \
+		$(BUILD_WORK)/arpoison/arpoison.o \
+		-L$(BUILD_WORK)/linking/ \
+		-l net
```

Comment:
Use `LDFLAGS` and remove `-L$(BUILD_WORK)/linking`, also dynamically link libnet, which adding `LDFLAGS` should do automatically.